assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -69,13 +84,14 @@ const ctaLinks = {
 // ----- Functions ----- //
 
 const getContribAttrs = ({
-  contribType, contribAmount, intCmp, showMonthly,
+  contribType, contribAmount, intCmp, showMonthly, isoCountry,
```

Comment:
Can you add flow types to these parameters?  Is isoCountry guaranteed not to be `null`?